speedtest-go.yaml

```diff
@@ -0,0 +1,28 @@
+package:
+  name: speedtest-go
+  version: 1.6.10
+  epoch: 0
+  description: CLI and Go API to Test Internet Speed using speedtest.net
+  copyright:
+    - license: MIT
+
+pipeline:
+  - uses: go/install
+    with:
+      package: github.com/showwin/speedtest-go
+      version: v${{package.version}}
```

Comment:
Can we do `git-checkout` and `go-build` instead to allow for gobumps when needed? 🤔